persistence/src/main/java/org/cbioportal/genome_nexus/persistence/internal/SourceInfoRepository.java

```diff
@@ -0,0 +1,41 @@
+package org.cbioportal.genome_nexus.persistence.internal;
+
+import org.cbioportal.genome_nexus.model.AggregateSourceInfo;
+import org.cbioportal.genome_nexus.model.GenomeNexusInfo;
+import org.cbioportal.genome_nexus.model.VEPInfo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class SourceInfoRepository {
+
+    @Autowired
+    private GenomeNexusInfo genomeNexusInfo;
+
+    @Autowired
+    private VEPInfo vepInfo;
+
+    @Autowired
+    private AggregateSourceInfo aggregateSourceInfo;
+
+    @Bean
```

Comment:
good .. as beans these will only be initialized a single time during startup and will then be reused by reference throughout the server execution.